### PR TITLE
Adds RN batch to 4.8

### PR DESCRIPTION
**Pull request by stevsmit**
No BZ.

Preview:

Last big batch!

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -1517,14 +1528,37 @@ The `authentication` and `openshift-apiserver` Operators now ignore the `oauth-a
 ** link:https://bugzilla.redhat.com/show_bug.cgi?id=1924585[*BZ#1924585*]
 ** link:https://bugzilla.redhat.com/show_bug.cgi?id=1924747[*BZ#1924747*]
 ** link:https://bugzilla.redhat.com/show_bug.cgi?id=1925083[*BZ#1925083*]
-//section 2
+
+* Previously, the web console relied on hard-coded channel strings to populate the channel modal dropdown. As a result, users could see channel values that may not be correct for their current version. Now, if the Cluster Version Operator does not supply the correct channels for a given version, the channel modal dropdown changes to a text input field and suggests channels and help text for the user. The console no longer relies on hard coded channel string. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1932281[*BZ#1932281*])
+
+* Previously, timestamps were not correctly formatted for Chinese or Japnaese languages. As a result, timestamps were harder to read, which provided a bad user experience. With this update, default timestamp formats are used for Chinese and Japanese in Moment.js, which provides a better user experience. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1932453[*BZ#1932453*])
```
Comment (bmcelvee) [inline release_notes/ocp-4-8-release-notes.adoc:1534]: Should Moment.js be in backticks?

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -1517,14 +1528,37 @@ The `authentication` and `openshift-apiserver` Operators now ignore the `oauth-a
...
+
+* Previously, the `rowFilters` prop in the FilterToolbar component did not accept the `null` value. So if the `rowFilters` prop was undefined, the uncaught exception was thrown. Now, when the `rowFilters` prop is referenced in the FilterToolbar component, the `null` value is accepted. As a result, the FilterToolbar does not throw exceptions when rowFilters prop is undefined. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1937018[*BZ#1937018*])
```
Comment (bmcelvee) [inline release_notes/ocp-4-8-release-notes.adoc:1536]: Missing backticks on the last `rowFilters`

File: release_notes/ocp-4-8-release-notes.adoc [hunk 3]
```diff
@@ -1517,14 +1528,37 @@ The `authentication` and `openshift-apiserver` Operators now ignore the `oauth-a
...
 
 * The fix for link:https://bugzilla.redhat.com/show_bug.cgi?id=1953097[BZ#1953097] enabled the CoreDNS `bufsize` plug-in with a size of 1232 bytes. Some primitive DNS resolvers are not capable of receiving DNS response messages over UDP that are greater than 512 bytes. Consequently, some DNS resolvers, such as Go’s internal DNS library, are unable to receive verbose DNS responses from the DNS Operator. This update sets the CoreDNS `bufsize` to 512 bytes for all servers. As a result, UDP DNS messages are now properly received. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966116[*BZ#1966116*])
 
+* Previously, cluster upstream resolver returned DNS responses that exceeded 512 bytes via UDP. Consequently, coreDNS returned `SERVFAIL` or other error messages and forced the client to retry over TCP. This update enabled the coreDNS bufisze plugin with a UDP buffer size of 1232 bytes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949361[*BZ#1949361*]
+
```
Comment (bmcelvee) [inline release_notes/ocp-4-8-release-notes.adoc:1560]: Missing end parentheses:

(link:https://bugzilla.redhat.com/show_bug.cgi?id=1949361[*BZ#1949361*])

File: release_notes/ocp-4-8-release-notes.adoc [hunk 4]
```diff
@@ -1593,6 +1627,24 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=19485
...
 
+* Previously, the installer did not take into account `defaultMachineSet` values in the install config during its validation which caused the installer to fail. This update applies the default values to the install config and starts validating empty fields.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1903055[*BZ#1903055*])
+
+* Previously, soft-anti-affinity required the client to set a minimum Nova microversion. Most versions of Ansible OS server module did not automatically require the client to set a minimum. As a result, the soft-anti-affinity commands were likely to fail. This update fixes the use of the Python OpenStack client to set a Nova microversion when dealing with soft-anti-affinity. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1910067[*BZ#1910067*])
```
Comment (bmcelvee) [inline release_notes/ocp-4-8-release-notes.adoc:1632]: Suggestion: I think `soft-anti-affinity` should be in backticks.

File: release_notes/ocp-4-8-release-notes.adoc [hunk 5]
```diff
@@ -1621,7 +1673,8 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=19485
 
 *Web console (Administrator perspective)*
 
-//section 1
+* Previously, when attempting to delete a custom resource within the CNV namespace in the console UI in developer mode, clicking *Delete* in resulted in the *Delete* button hanging in a stuck state. Additionally, an error message that appears when performing the same action in the CLI was not displaying. With this update, the error message displays as expected and the *Delete* button does not stick. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939753[*BZ#1939753*]).
+
```
Comment (bmcelvee) [inline release_notes/ocp-4-8-release-notes.adoc:1677]: Extra period after (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939753[*BZ#1939753*]).

File: release_notes/ocp-4-8-release-notes.adoc [hunk 6]
```diff
@@ -2313,6 +2366,9 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=18786
 
 * Previously, the topology URLs created for deployments using Bitbucket repository in the {product-title} web console did not work if they included a branch name that contained a slash character. This was due to an issue with the Bitbucket API (link:https://jira.atlassian.com/browse/BCLOUD-9969[BCLOUD-9969]). The current release mitigates this issue. If a branch name contains a slash, the topology URLs point to the default branch page for the repository. This issue will be fixed in a future release of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1970470[*BZ#1970470*].
 
+* Installing {product-title} (OCP) version 4.6 on {rh-virtualization-first} requires RHV version 4.4. If you are running an earlier version of OCP on RHV 4.3, do not update it to OCP version 4.6. Red Hat has not tested running OCP version 4.6 on RHV version 4.3 and does not support this combination. For additional information about tested integrations, see https://access.redhat.com/articles/4763741
+
```
Comment (bmcelvee) [inline release_notes/ocp-4-8-release-notes.adoc:2370]: Link formatting missing: 

see link:https://access.redhat.com/articles/4763741[OpenShift Container Platform 4.x Tested Integrations (for x86_x64)].

**Comment by bmcelvee** [comment]
Just a few suggestions/edits. All set!

**Review by bobfuru — APPROVED** [review]

**Pull request merged**